packages/untp-test-suite/src/core/utils/common.ts

```diff
@@ -29,6 +15,13 @@ export const readFile = async <T>(filePath: string): Promise<T> => {
   return JSON.parse(fileContent);
 };
 
+export const readConfigFile = async () => {
+  const configPath = path.resolve(process.cwd(), '../../config/credentials.json');
+  const result = await readFile<ConfigCredentials>(configPath);
+  validateCredentialConfigs(result.credentials);
```

Comment:
To make the `readConfigFile` function easier to understand, I suggest it should only do what its name implies: read the configuration file. The `validateCredentialConfigs` should be called where we call the `readConfigFile` function. This approach follows the principle of clean code, which emphasizes that each function should have a single, well-defined purpose.